jsonexample/test/simple_object_unit_test.dart

```diff
@@ -0,0 +1,196 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter_test/flutter_test.dart';
+import 'package:jsonexample/built_value/built_simple_object.dart';
+import 'package:jsonexample/built_value/built_value_serializers.dart';
+import 'package:jsonexample/dart_convert/converted_simple_object.dart';
+import 'package:jsonexample/json_serializable/serializable_simple_object.dart';
+
+void main() {
+  const Map<String, dynamic> typicalObjectJson = {
```

Comment:
[nit] I think the Dart style, if you want to type-annotate, is:

```
const typicalObjectJson = <String,dynamic>{
  // ...
}
```

And in this particular case, you don't even need to do that, since it's going to be inferred.

Types on the left are encourages for top-level members, but local variables tend to rely on inferred types.
https://www.dartlang.org/guides/language/effective-dart/design#prefer-type-annotating-public-fields-and-top-level-variables-if-the-type-isnt-obvious

If the second parameter wasn't dynamic, you could just leave generics altogether. But because of [this rule](https://www.dartlang.org/guides/language/effective-dart/design#prefer-annotating-with-dynamic-instead-of-letting-inference-fail), you want to use it.

Reply:
Done.